app/src/main/java/com/yagni/view/ManageInvGUI.java

```diff
@@ -9,365 +9,447 @@
-        private static JButton backButton = new JButton("Back");
-        private static JButton logoutButton = new JButton("Logout");
-        private static JButton clearButton = new JButton("Clear");
-       
-        private static JTextField productIDField = new JTextField(20);
-        private static JTextField quantityField = new JTextField(20);
-        private static JTextField wholeSaleField = new JTextField(20);
-        private static JTextField salePriceField = new JTextField(20);
-        private static JTextField supplierIDField = new JTextField(20);
-        private static JTextField searchField = new JTextField(20);
-        
-        ManageInvGUI(DbConnection linkDBIn){
-            
-            // setting the connection with database
-        	linkDB = linkDBIn;
-
-            // Importing and setting custom font Caveat for all text components 
-            try {
-                File font_file = new File("App/assets/fonts/Caveat-VariableFont_wght.ttf");
-                Font caveatFont = Font.createFont(Font.TRUETYPE_FONT, font_file).deriveFont(20f);
-                GraphicsEnvironment ge = GraphicsEnvironment.getLocalGraphicsEnvironment();
-                ge.registerFont(Font.createFont(Font.TRUETYPE_FONT, font_file));
-                headerLabel.setFont(caveatFont.deriveFont(Font.BOLD, 35f));
-                controlsTitle.setFont(caveatFont.deriveFont(Font.BOLD, 25f));
-                addButton.setFont(caveatFont);
-                productIDLabel.setFont(caveatFont);
-                productActionsLabel.setFont(caveatFont);
-                quantityLabel.setFont(caveatFont);
-                wholeSaleLabel.setFont(caveatFont);
-                salePriceLabel.setFont(caveatFont);
-                supplierIDLabel.setFont(caveatFont);
-                updateButton.setFont(caveatFont);
-                deleteButton.setFont(caveatFont);
-                loadButton.setFont(caveatFont);
-                reloadButton.setFont(caveatFont);
-                searchField.setFont(caveatFont);
-                backButton.setFont(caveatFont);
-                logoutButton.setFont(caveatFont);
-                clearButton.setFont(caveatFont);
-                footerLabel.setFont(caveatFont.deriveFont(16f));	    
-            } catch (IOException | FontFormatException e) {
-                e.printStackTrace();
-            }
+public class ManageInvGUI implements ActionListener, FocusListener, MouseListener {
+
+    private static JFrame inventoryFrame; // Jframe to display all elements
+    private static JTable productsTable; // JTable to display all products
+    private static JScrollPane tableScroll; // JScrollPane to scroll through the products JTable
+    private static DbConnection linkDB; // Variable to persist connection to db
+
+    // Variable declarations for all JPanels needed within GUI
+    private static JPanel headerPanel, backPanel, logoutPanel, bodyPanel, footerPanel, tablePanel, controlsPanel;
```

Comment:
For JPanel, JLabel, JButton, JTextField, ImageIcon I would declare each one on it's own line instead of grouping them all in one line.

4.6.1 & 4.8.2 mentions how it should look

I can explain it better in discord chat later today when you have time.

I would still group JPanel variables together w/o whitespace in-between similar variables, and white space between the next grouping. Such as, a whitespace separating buttons vs. panels. But for each variable to have it's own line.

Reply:
Variable declarations have been changed to reflect the style guide. Apologies I missed 4.8.2 somehow!